### PR TITLE
feat: added Slider and responsive navigation bar 

### PR DESCRIPTION
**Pull request by Chakravarthi0**
Added a slider and responsive navbar

### CODE REVIEW

File: components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.slider-input::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    width: 2rem;
+    height: 2rem;
+    background-color: var(--white);
+    border: 2px solid var(--primary-color);
+    cursor: pointer;
+    box-shadow: 0 0 0 1px var(--white);
```
Comment (mimansha-swarup) [inline components/slider/slider.css:39]: make variable for box shadows too

**Review by mimansha-swarup — APPROVED** [review]
except that all looks good to me

**Review by srishti-maurya — APPROVED** [review]
looks good

**Comment by KaustubhM1997** [comment]
Looks good!
Suggestion: A hover over the navbar elements might look better rather than changing the color altogether when clicked.

**Comment by Chakravarthi0** [comment]
> Looks good! Suggestion: A hover over the navbar elements might look better rather than changing the color altogether when clicked.

Thanks for the the suggestion Kaustubh. I will work on it.

**Pull request merged**